modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/throttling/NewThrottleTiersAndLoadOldTiersTestCase.java

```diff
@@ -0,0 +1,610 @@
+                stringBuilder.append(line);
+            }
+            return stringBuilder.toString();
+        } catch (IOException ioE) {
+            throw new APIManagerIntegrationTestException("IOException when reading the file from:" + fileLocation, ioE);
+        } finally {
+            if (bufferedReader != null) {
+                try {
+                    bufferedReader.close();
+                } catch (IOException e) {
+                    log.warn("Error when closing the buffer reade which used to reed the file:" + fileLocation +
+                            ". Error:" + e.getMessage());
+                }
+            }
+        }
+    }
+
+    protected HttpResponse subscribeToAPI(APIIdentifier apiIdentifier, String applicationName,
+            APIStoreRestClient storeRestClient) throws APIManagerIntegrationTestException {
+        SubscriptionRequest subscriptionRequest = new SubscriptionRequest(apiIdentifier.getApiName(),
+                apiIdentifier.getProviderName());
+        subscriptionRequest.setVersion(apiIdentifier.getVersion());
+        subscriptionRequest.setApplicationName(applicationName);
+        if ((apiIdentifier.getTier() != null) && (!apiIdentifier.getTier().equals(""))) {
+            subscriptionRequest.setTier(apiIdentifier.getTier());
+        }
+        return storeRestClient.subscribeToAPI(subscriptionRequest);
+    }
+
+    /**
+     * Generate the access token
+     *
+     * @param storeRestClient - Instance of storeRestClient
+     * @param applicationName - Application name
+     * @return ApplicationKeyBean - ApplicationKeyBean that contains access token, consumer key and consumer secret
+     * @throws APIManagerIntegrationTestException - Exception throws by the  method call of generateApplicationKey()
+     *                                            in APIStoreRestClient.java
+     */
+
+    protected ApplicationKeyBean generateApplicationKeys(APIStoreRestClient storeRestClient, String applicationName)
+            throws APIManagerIntegrationTestException {
+
+        try {
+            ApplicationKeyBean applicationKeyBean = new ApplicationKeyBean();
+            APPKeyRequestGenerator generateAppKeyRequest = new APPKeyRequestGenerator(applicationName);
+            String responseString = storeRestClient.generateApplicationKey(generateAppKeyRequest).getData();
+            JSONObject response = new JSONObject(responseString);
+
+            applicationKeyBean.setAccessToken(response.getJSONObject("data").getJSONObject("key").
+                    get("accessToken").toString());
+            applicationKeyBean.setConsumerKey(response.getJSONObject("data").getJSONObject("key").
+                    get("consumerKey").toString());
+            applicationKeyBean.setConsumerSecret(response.getJSONObject("data").getJSONObject("key").
+                    get("consumerSecret").toString());
+            return applicationKeyBean;
+        } catch (Exception e) {
```

Comment:
already revert changes in commit: https://github.com/wso2/product-apim/commit/a75c4880b08d984d9a0e971387c0d6a07ef2c886